modules/packages/ZMQ.chpl

```diff
@@ -913,7 +926,33 @@ module ZMQ {
         }
       }
     }
+    
+    proc parallelCreateBytesWithNewBuffer(x: c_ptr(?t), length: int, size=length+1) {
```

Comment:
I think we want to make this a `private proc`, both so that other modules can't call it and so we don't document it in the generated chpldoc.